pkg/cm_accounts/cm_accounts.go

```diff
@@ -20,11 +22,18 @@ import (
 	"go.uber.org/zap"
 )
 
+const (
+	// Implementation slot for ERC1967Proxy
+	// See: https://eips.ethereum.org/EIPS/eip-1967#logic-contract-address
+	implementationSlotString = "0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc"
+)
+
 var (
 	_ Service = &service{}
 
 	bigZero            = big.NewInt(0)
 	chequeOperatorRole = crypto.Keccak256Hash([]byte("CHEQUE_OPERATOR_ROLE"))
+	implementationSlot = common.HexToHash(implementationSlotString)
```

Comment:
```suggestion
	cmManagerImplementationSlot = common.HexToHash(cmManagerImplementationSlotString)
```

Reply:
actually, better managerCMImplementationSlot - that way its more clear structuraly, that its manager's cm impl slot